packages/docs/docs/guides/3-Portal-API/1-create-and-share-request.md

```diff
@@ -19,7 +19,7 @@ We assume you know React. Before you run the code below, don't forget to:
 - Change the `API_KEY`, with your test API key
 - Change the payment value with one of your Ethereum addresses
 
-Even if for this tests the transaction will be done over the Rinkeby network, it's more realistic to configure your own address.
+Even if for this tests the transaction will be done over the Goerli network, it's more realistic to configure your own address.
```

Comment:
a bit early to modify all docs I think. The point of this PR is to _enable_ Goerli, not yet make it the default